egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -145,8 +132,9 @@ def compile(self):
                 "pdflatex -interaction=batchmode "
                 "-output-directory={0} {1}".format(dir_name, latex_file))
         except Exception as e:
-            logger.warning("There was an error compiling the latex file {0}, "
-                           "please do it manually: {1}".format(latex_file, e))
+            logger.warning("There was an error compiling latex file {0}. "
+                           "Check report.log generated by pdflatex in the same directory. "
+                           "{1}".format(latex_file, e))
```

Comment:
Could you modify this string to use "%s" specification similar to the other modifications you made.

Reply:
Done, in all logger.xxx invocations, I believe.